[2022-02-10 17:27:51,185] {taskinstance.py:1026} INFO - Dependencies not met for <TaskInstance: green_taxi_download_ingest_2_gcs_dag.local_to_gcs_green scheduled__2019-01-02T06:00:00+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_tasks_state={'total': 1, 'successes': 0, 'skipped': 0, 'failed': 0, 'upstream_failed': 0, 'done': 0}, upstream_task_ids={'format_2_parquet_task'}
[2022-02-10 17:27:51,190] {local_task_job.py:99} INFO - Task is not able to be run
[2022-02-10 17:28:18,537] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: green_taxi_download_ingest_2_gcs_dag.local_to_gcs_green scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-10 17:28:18,660] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: green_taxi_download_ingest_2_gcs_dag.local_to_gcs_green scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-10 17:28:18,667] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 17:28:18,671] {taskinstance.py:1239} INFO - Starting attempt 3 of 3
[2022-02-10 17:28:18,673] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 17:28:18,831] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_green> on 2019-01-02 06:00:00+00:00
[2022-02-10 17:28:18,857] {standard_task_runner.py:52} INFO - Started process 647 to run task
[2022-02-10 17:28:18,872] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'green_taxi_download_ingest_2_gcs_dag', 'local_to_gcs_green', 'scheduled__2019-01-02T06:00:00+00:00', '--job-id', '592', '--raw', '--subdir', 'DAGS_FOLDER/greentaxi_download_2_gcs.py', '--cfg-path', '/tmp/tmpwh4iz1tq', '--error-file', '/tmp/tmpyh_9szb_']
[2022-02-10 17:28:18,878] {standard_task_runner.py:77} INFO - Job 592: Subtask local_to_gcs_green
[2022-02-10 17:28:19,271] {logging_mixin.py:109} INFO - Running <TaskInstance: green_taxi_download_ingest_2_gcs_dag.local_to_gcs_green scheduled__2019-01-02T06:00:00+00:00 [running]> on host 2107cbac5c9b
[2022-02-10 17:28:19,539] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 17:28:19,650] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=green_taxi_download_ingest_2_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_green
AIRFLOW_CTX_EXECUTION_DATE=2019-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-02T06:00:00+00:00
[2022-02-10 17:28:25,021] {python.py:175} INFO - Done. Returned value was: None
[2022-02-10 17:28:25,094] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=green_taxi_download_ingest_2_gcs_dag, task_id=local_to_gcs_green, execution_date=20190102T060000, start_date=20220210T172818, end_date=20220210T172825
[2022-02-10 17:28:25,206] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 17:28:25,310] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
